files/app/main/status/e/basics.ut

```diff
@@ -242,7 +249,7 @@ if (request.env.REQUEST_METHOD === "DELETE") {
             <div class="cols">
                 <div>
                     <div class="o">Upload SSH Key</div>
-                    <div class="m">Add SSH key</div>
+                    <div class="m">Add SSH key {% if (!detectEd25519()) { %} (no Ed25519 support) {% } %}</div>
```

Comment:
Let's not do that here. If you look a little lower in the file you'll find some inline help. If you want to add something there there about what keys are supported in both cases I would prefer that. I think telling someone what is supported rather than telling them what isn't would be better messaging.

Reply:
That is a good idea.